Block/Checkout/Success.php

```diff
@@ -44,22 +44,30 @@ class Success extends \Magento\Framework\View\Element\Template
      */
     protected $_orderFactory;
 
+    /**
+     * @var \Magento\Framework\Pricing\Helper\Data
+     */
+    public $priceHelper;
```

Comment:
Where do you use this public priceHelper?

Reply:
In the template file, in order to format the value that we get in the response using the Magento configured locale https://github.com/Adyen/adyen-magento2/pull/656/files#diff-b47a2330059e98c2cdb5ce9b576ab116R120